### ISSUE TITLE
mingw-w64-x86_64-npm package not present

### ISSUE THREAD

**Issue by TBBle** [Alexpux/MINGW-packages#2249]
In https://github.com/Alexpux/MINGW-packages/commit/2a7b363e05507b86d8ba520ed6b5a43c7987f732 npm was broken out of nodejs as a separate package.

Today I updated to `mingw-w64-x86_64-nodejs-6.9.5-1`, and it now correctly has an optional dependency on `mingw-w64-x86_64-npm`.

However, there is no `mingw-w64-x86_64-npm` package, even though the source clearly exists: https://github.com/Alexpux/MINGW-packages/tree/master/mingw-w64-npm

In case the expectation is that either tea-ci or Appveyor will auto-upload new packages to make them available, both failed:
* https://tea-ci.org/Alexpux/MINGW-packages/2447 
````
==> Starting build()...
input is not a tty
==> ERROR: A failure occurred in build().
    Aborting...
````

* https://ci.appveyor.com/project/Alexpux/mingw-packages/build/1.0.2413
````
Deploying using BinTray provider
Uploading "ci.mingw.builds" to BinTray as alexpux/msys2/ci.mingw/latest/ci.mingw.builds...Error uploading artifact to BinTray (403 status): Version 'latest' has been published more than 180 days ago and can no longer be deployed to.
````

**Comment by mati865** [comment]
It doesn't build on real machine for unknown reason.

**Comment by TBBle** [comment]
If that's the apparent hang on the line `node cli.js install ../npm-marked-man-${_marked_man_ver}.tgz` I can confirm that it happens after npm attempts to return success, using `MINGW_INSTALLS=mingw64 makepkg-mingw -sLfC`

**Comment by mati865** [comment]
> If that's the apparent hang on the line node cli.js install ../npm-marked-man-${_marked_man_ver}.tgz 

Workaround for it is to add `sleep 5` before because previous node command doesn't exit fast enough.

More serious problem is here: https://github.com/Alexpux/MINGW-packages/pull/2211#issuecomment-284254626

**Comment by TBBle** [comment]
`sleep 5` didn't work around the problem for me. I didn't play around much harder than that. `sleep 30` also didn't work. It sounds like the problem is more general though.

**Comment by leachim6** [comment]
I'm also having the same issue when building this package

___
 $ MINGW_INSTALLS=mingw64 makepkg-mingw -sLf
==> Making package: mingw-w64-npm 4.3.0-1 (Thu, Mar 23, 2017  4:25:40 AM)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found npm-4.3.0.tar.gz
  -> Found npm-marked-man-0.2.0.tgz
  -> Found npm-marked-0.3.6.tgz
==> Validating source files with sha256sums...
    npm-4.3.0.tar.gz ... Passed
    npm-marked-man-0.2.0.tgz ... Passed
    npm-marked-0.3.6.tgz ... Passed
==> Extracting sources...
==> Starting prepare()...
==> Removing existing $pkgdir/ directory...
==> Starting build()...
npm@4.3.0 C:\msys64\home\leachim6\src\npm\src\npm-4.3.0
`-- marked@0.3.6

npm@4.3.0 C:\msys64\home\leachim6\src\npm\src\npm-4.3.0
`-- marked-man@0.2.0
___

Any word on a workaround?

**Comment by andrebreves** [comment]
I was able to build when I didn't use the log -L option: `makepkg-mingw -sf`

**Comment by TBBle** [comment]
Interesting. Suggests it's a stdio issue? When I was trying to make it work before, I played with `winpty` to see if it helped, but it didn't seem to make a difference.

That said, I believe there's an alias in one of the bash startup files which automatically applies `winpty` to `node` invocations. It probably wouldn't affect the command run from the build script though.

**Comment by mati865** [comment]
I'm building almost everything without `-L` and it doesn't help.

BTW `-L` is recommend when you want to attach logs when reporting issues with build.

**Comment by ItachiSan** [comment]
I do have the same problem, though I seem able to build the package.

**Comment by BMBurstein** [comment]
What do I have to do to get npm? I didn't understand the workarounds suggested. (Add `sleep 5` to where? Something with `log -L`?)

**Comment by mati865** [comment]
Npm is included in nodejs package for now.
You might want to test it.

**Comment by TBBle** [comment]
I had to ```rm -rf /mingw64/lib/node_modules/npm`` but it's likely that's because either I'd hand-installed NPM at some point to work around this issue, or the files had been left behind by a successful hand-build of this npm package. I looks like I had npm 4 installed (or parts thereof) so I don't think it will affect anyone who was simply bitten by and blocked by this issue.

Either way, it appears upgrading to ```mingw-w64-x86_64-nodejs``` after https://github.com/Alexpux/MINGW-packages/commit/60d851100774766701456e358c93e49f77bd5e58 has resolved the issue (leaving me back at npm 3.10.10 per ``npm --version``) so I'll close this ticket.

However, the recipe for ```mingw-w64-x86_64-npm``` still exists, and is presumably still unbuildable.
